static/styles/rendered_markdown.scss

```diff
@@ -552,3 +552,39 @@ a:hover code {
 .highlight {
     background-color: hsl(51, 94%, 74%);
 }
+
+.rendered_markdown pre {
+    &:hover .copy_codeblock,
+    &:focus-within .copy_codeblock {
+        visibility: visible;
+    }
+}
+
+/* TODO: Consolidate with .copy_message */
+/* Style copy-to-clipboard button inside codeblocks */
+.copy_codeblock,
+.copy_codeblock:focus {
+    visibility: hidden;
+    outline-color: hsl(0, 0%, 73%);
+    height: 18px;
+    width: 10px;
+    padding: 6px 6px;
+    /* Having absolute positioning here ensures that the element
+    doesn't scroll along with the code div in narrow windows */
+    position: absolute;
+    right: 0;
+    margin-top: -6px;
+    display: block;
+    /* The below two avoids the padded element from displaying
+    it's own border and background color */
+    border: none;
+    background-clip: content-box;
+}
+
+.copy_codeblock:hover svg path {
+    fill: hsl(200, 100%, 40%);
+}
+
+.preview_content .copy_codeblock {
+    display: none;
+}
```

Comment:
Should these all be scoped inside `.rendered_markdown`?  I would think so, since structurally the elements are only created when processing rendered markdown.

Reply:
Yup, moved them :+1: